src/test/scala/uk/gov/hmcts/ccd/util/CcdTokenGenerator.scala

```diff
@@ -1,27 +1,83 @@
 package uk.gov.hmcts.ccd.util
 
-import org.springframework.http.HttpMethod
+import java.net.{URL, URLDecoder}
+import java.util
+import spray.json._
+import spray.json.DefaultJsonProtocol._
+
+import org.springframework.http.{HttpEntity, HttpHeaders, MediaType}
 import org.springframework.web.client.RestTemplate
 import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator
+import org.springframework.util.LinkedMultiValueMap
+import org.springframework.util.MultiValueMap
 
 object CcdTokenGenerator extends PerformanceTestsConfig with SpringApplicationContext {
 
-  val TOKEN_LEASE_URL = s"$UserAuthUrl/testing-support/lease"
+  val AUTHORISATION_CLIENT_ID = s"ccd_gateway"
```

Comment:
in this case we can simply have "ccd_gateway"

Reply:
done